docs/psoc6/feature_list.rst

```diff
@@ -121,8 +124,6 @@ Table :ref:`configuration details <table_mpy_configuration>` below lists specifi
 +-----------------+----------------------------------------------------------------------------------------------------------------------+                                                                                                                                             
 | machine.SoftSPI | Option ``MICROPY_PY_MACHINE_SOFTSPI`` enabled.                                                                       |
 +-----------------+----------------------------------------------------------------------------------------------------------------------+                                                                                                                                             
-| machine.Timer   | mode = Timer.PERIODIC is not supported                                                                               |
```

Comment:
Mmm, this has been removed accidentally.

Reply:
Fixed